packages/quantic/force-app/main/translations/fr.translation-meta.xml

```diff
@@ -600,4 +600,40 @@
       <label>Ouvre dans un onglet du navigateur web</label>
       <name>quantic_OpensInBrowserTab</name>
     </customLabels>
+    <customLabels>
+        <label>Erreur du composant {{0}}</label>
+        <name>quantic_ComponentError</name>
+    </customLabels>
+    <customLabels>
+        <label>Consultez la console développeur pour plus d'informations.</label>
+        <name>quantic_LookAtDeveloperConsole</name>
+    </customLabels>
+    <customLabels>
+        <label>"{{0}}" est un nombre maximum de suggestions de documents non valide. Un nombre entier supérieur à 0 était attendu.</label>
+        <name>quantic_InvalidMaxNumberOfDocumentSuggestions</name>
+    </customLabels>
+    <customLabels>
+        <label>"{{0}}" n'est pas un nombre valide de suggestions de documents ouvertes automatiquement. Un nombre entier positif était attendu.</label>
+        <name>quantic_InvalidNumberOfAutoOpenedDocuments</name>
+    </customLabels>
+    <customLabels>
+        <label>Le nom API du champ Salesforce "{{0}}" est introuvable.</label>
+        <name>quantic_SalesforceFieldNotFound</name>
+    </customLabels>
+    <customLabels>
+        <label>"{{0}}" est un nombre maximum de suggestions non valide. Un nombre entier positif était attendu.</label>
+        <name>quantic_InvalidMaxNumberOfSuggestions</name>
+    </customLabels>
+    <customLabels>
+        <label>La propriété "{{0}}" est obligatoire, veuillez définir sa valeur.</label>
```

Comment:
@mmitiche If EN suggestion is taken, FR equivalent is: 
"Il faut saisir une valeur pour la propriété "{{0}}"."